### PR TITLE
Create mobile version layout solicitud

### PR DESCRIPTION
**Pull request by Andresbl123**
The mobile version of the application is created, where the presentation of each of the sections that are part of this page is adjusted, this PR is limited only to the layout, so for the moment in terms of functionality some adjustments still need to be made, 

Note: in this branch and settings, the use cases of the cb-64 and cb-66 cards are realised.

### CODE REVIEW

File: src/pages/board/outlets/financialReporting/index.tsx
Comment (novadeveloper14) [inline src/pages/board/outlets/financialReporting/index.tsx:1]: the component is getting corrupted in its desktop version

image
image

Comment (novadeveloper14) [inline src/pages/board/outlets/financialReporting/index.tsx:1]: It seems that from a point here there should be a separation
image

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]

File: src/pages/board/outlets/financialReporting/index.tsx
Comment (Andresbl123) [reply src/pages/board/outlets/financialReporting/index.tsx:1]: resolved

Comment (Andresbl123) [reply src/pages/board/outlets/financialReporting/index.tsx:1]: resolved

**Review by novadeveloper14 — APPROVED** [review]

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]
Some size or type of the text components of this card and margin/padding change in its mobile version, please solve so that the requirement of the cb66 task can be met

App:
image

Figma:
image

image

**Comment by novadeveloper14** [comment]
Please, if the PR solves more tasks besides the main branch, please let me know in the description of the PR.

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]
comment related to task: cb64

The ToDo component already has its mobile version available, so we should pass the isMobile variable to it from the index.tsx of FinancialReporting:

image

After this, adjust the separation of the cards with the ends of the screen since it is becoming a little out of adjustment.

App:
image

Figma:

image

**Review by novadeveloper14 — APPROVED** [review]

**Review by carin91 — APPROVED** [review]

**Pull request merged**